packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -220,21 +222,21 @@ contract FastBridgeV2 is Admin, IFastBridgeV2 {
         bytes32 transactionId = keccak256(request);
         BridgeTransaction memory transaction = getBridgeTransaction(request);
 
-        // update bridge tx status if able to claim origin collateral
-        if (bridgeStatuses[transactionId] != BridgeStatus.RELAYER_PROVED) revert StatusIncorrect();
+        BridgeTxDetails storage _thisBridgeTxDetails = bridgeTxDetails[transactionId];
```

Comment:
Not a huge fan of this tbh, makes the code more complex without a particularly good reason. Could either read the full proof in memory (as before), or unpack only the required variables (status, proofBlockTimestamp, proofRelayer) onto the stack as in the comment above.